src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsDetailsMapping.java

```diff
@@ -212,4 +214,15 @@ public static boolean isPoOfficerAttending(@Valid SscsCaseData caseData) {
 
         return isYes(caseData.getDwpIsOfficerAttending());
     }
+
+    public static String adjournCaseInterpreterLanguage(@Valid SscsCaseData caseData) {
+        if (isNull(caseData.getAdjournCaseInterpreterRequired())) {
+            return caseData.getLanguagePreference().getCode();
+        }
+        if (isYes(caseData.getAdjournCaseInterpreterRequired()) && nonNull(caseData.getAdjournCaseInterpreterLanguage())) {
+            return caseData.getAdjournCaseInterpreterLanguage();
+        }
+        return "no interpreter required";
```

Comment:
Magic string - should this not be a constant? And checking that this is what is actually required to be returned

Reply:
Made string into a constant. 
When demoing this to Makeda, she was happy with this behaviour.